### DISCUSSION TITLE
sms apk For an Android device, keys, that is, it does not have the option of touch

### DISCUSSION THREAD

**Discussion by rgliR** [community/community#124876] [Programming Help]
### Select Topic Area

Question

### Body

Hi
I'm looking for an app for SMS messages (apk is better)
For an Android device, keys, that is, it does not have the option of touch
I'm a fairly new user here so I don't exactly understand how it works here
I would very much appreciate an answer
Thank you very much to all who answered

**Comment by LiteBrite82** [comment]
Hi @rgliR ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project.

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
